R/modelFitting.R

```diff
@@ -503,17 +521,16 @@ compBayes <- function(dat,
   for(i in 2:5){
```

Comment:
so `2:5` are hard-coded for particular columns? What happens if we add or remove a column, or re-order them?